### PR TITLE
Feature/gh 11537 conflict handling fixed

### LINKED ISSUE #11537: check  CCV2 CY Tests failures and fix or create follow up tickets
http://cpqjenkins.mo.sap.corp:8080/view/CCV2_CPQ/job/CCV2_CPQ_CYPRESS_TESTS/

### PR DESCRIPTION
**Pull request by Changsuwan**

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts [hunk 1]
```diff
@@ -178,8 +183,10 @@ export function clickOnEditConfigurationLink(cartItemIndex: number): void {
 export function clickOnRemoveLink(cartItemIndex: number): void {
   cy.get('cx-cart-item-list .cx-item-list-row')
     .eq(cartItemIndex)
+    .not('disabled')
     .find('.cx-remove-btn')
     .within(() => {
+      cy.log('One cart item will bbe removed under index: ' + cartItemIndex);
```
Comment (Larisa-Staroverova) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts]: Type: **be** removed

File: projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts [hunk 2]
```diff
@@ -84,8 +84,8 @@ export function checkNotificationBannerOnOP(
   element,
   numberOfIssues?: number
 ): void {
-  element
-    .get('.cx-error-msg')
+  //element //Why use element instead of cy?
```
Comment (Larisa-Staroverova) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts]: Could you revert your changes and try to understand what the method does? You cannot change the API. This method is used in another tests that will fail.
In your tests you could use `verifyNotificationBannerOnOP` method that verifies whether the issues banner is displayed and the number of issues are accurate

**Review by Larisa-Staroverova — CHANGES_REQUESTED** [review]

File: projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts [hunk 3]
```diff
@@ -4,9 +4,9 @@ import * as productSearch from '../../helpers/product-search';
 import * as cart from '../../helpers/cart';
 
 const POWERTOOLS = 'powertools-spa';
-const EMAIL = 'cpq03@sap.com';
+const EMAIL = 'cpq13@sap.com';
```
Comment (Larisa-Staroverova) [inline projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts]: Could you change the user's e-mail to cpq03?

File: projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts [hunk 4]
```diff
@@ -4,9 +4,9 @@ import * as productSearch from '../../helpers/product-search';
...
 const PASSWORD = 'welcome';
-const CPQ_USER = 'cpq03';
+const CPQ_USER = 'cpq13';
```
Comment (Larisa-Staroverova) [inline projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts]: Could you change the userl to cpq03?

**Review by Larisa-Staroverova — CHANGES_REQUESTED** [review]

File: projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts (hunk 3 above)
Comment (Changsuwan) [reply projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts]: Done

File: projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts (hunk 4 above)
Comment (Changsuwan) [reply projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts]: Done

File: projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts (hunk 1 above)
Comment (Changsuwan) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts]: Done

**Review by Larisa-Staroverova — APPROVED** [review]

**Pull request merged**